### ISSUE TITLE
Client should have an option to download files

### ISSUE THREAD

**Issue by afrubin** [VariantEffect/MaveTools#14]
The API client should have an option to retrieve the CSV files for a score set record.

**Comment by JiahuaQu** [comment]
Yes, we would also like to get the entire table with matched genes and variants as well as other remaining indices in the following colomns. In this way, we can easily know how many genes are recorded in this database and whether a long list of genes we are interested in are recorded in the database. That will be much better than retrieving the information by searching genes one by one. Thank you very much.
